nrich-registry/README.md

```diff
@@ -13,7 +13,7 @@ of overriding
 ## Setting up Spring beans
 
 To be able to use this library following configuration is required. If Jacksons `ObjectMapper` is not available it should also be defined and if `nrich-form-configuration` (provides client side
-validation of registry entities) libary is not on classpath then `RegistryDataFormConfigurationResolverService` bean is not needed. History requires `hibernate-envers` dependency and if history is not
+validation of registry entities) libary is not on classpath then `RegistryDataFormConfigurationMappingCustomizer` bean is not needed. History requires `hibernate-envers` dependency and if history is not
```

Comment:
`RegistryDataFormConfigurationResolverService` is still present in this file, in line 175.